src/test/resources/regressions/features/closures/closures-fail5-proofs.gobra

```diff
@@ -66,8 +66,7 @@ func test4() {
 
     //:: ExpectedOutput(type_error)
     proof cl implements pspec {
-        //:: ExpectedOutput(type_error)
-        x = 2 // assignment in ghost code to a non-ghost variable
+        x = 2
```

Comment:
The entire implementation proof used to be ghost. I changed this such that the block is non-ghost as otherwise we couldn't call a non-ghost function as part of the implementation proof. As a consequence, the assignment does no longer cause a ghost error. However, the implementation proof itself still fails as this assignment is not permitted due to the restrictions imposed on statements occurring in an implementation proof